cirq/google/engine/engine.py

```diff
@@ -692,3 +693,10 @@ def get_calibration(self, calibration_name: str) -> calibration.Calibration:
         response = self.service.projects().processors().calibrations().get(
             name=calibration_name).execute()
         return calibration.Calibration(response['data']['data'])
+
+    def sampler(self, processor_id: Union[str, List[str]],
```

Comment:
I'm going to continue to object to accepting a list of processor_id's. Please document the (surprising!) semantics in this wrapper method if people are adamant that we keep it

Reply:
I copied documentation from QuantumEngineSampler but you may object that it is too vague. It's the best I can do though because I don't actually know the semantics (i.e., which processor gets executed? Do they get tried one at a time from the first one? I don't know. Perhaps @Strilanc can answer).

Reply:
If many processors are supplied it will schedule on the first available.  

See #2075  for a discussion of this, I think we decided there to have two separate args?